newpost.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<link href="https://fonts.googleapis.com/css?family=Vollkorn" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<title>New Post</title>
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<header class="topnav">
+		<div class="topnav_content">
+			<div>
+				<h1><a href="index.html">StackOverflow-Lite</a></h1>
+			</div>
+		
+
+			<div class="topnav_options_container">
+				<ol class="profile_options">
+					<li class="profile">
+						<a href="profile.html" id="profile_icon"></a>
+					</li>
+				</ol>
+				<ol class="auth_options">
+					<li class="auth">
+						<a class="button_secondary" href="signin.html">Log In</a>
+						<a class="button_primary" href="signup.html">Sign Up</a>
+					</li>
+				</ol>
+			</div>
+		</div>
+	</header>
+
+	<div class="main_body">
+		<div class="options_container">
+			<h2>New Question</h2>
+		</div>
+
+		<div class="new_post_container">
+			<div class="new_post_title_container">
+				<textarea class="postTextArea" rows="1" maxlength="150" placeholder="Title"></textarea>
+				
+			</div>
+
+			<div class="new_post_body_container">
+				<textarea class="postTextArea" rows= "10" placeholder="Question body"></textarea>
+				
+			</div>
+
+			<div class="submit_options_container">
+				<div class="submit_options">
+					<button class="button_secondary">Cancel</button>
+					<button class="button_primary">Save</button>
+				</div>
+			</div>
+			
+		</div>
+
+		
+	</div>
+	
+</body>
+
+</html>
```

Comment:
Add a new line here

Reply:
the new line should be added where exactly? after line 65?

Reply:
done